lib/ASTGen/Sources/ASTGen/PluginHost.swift

```diff
@@ -319,7 +319,7 @@ class PluginDiagnosticsEngine {
     messageSuffix: String? = nil
   ) {
     for diagnostic in diagnostics {
-      self.emit(diagnostic)
+      self.emit(diagnostic, messageSuffix: messageSuffix)
```

Comment:
We had a bug in executable plugins where ` (from macro '\(macroName)')` suffix wasn't added in the diagnostics from executable plugins 😞